app/controllers/katello/api/v2/flatpak_remotes_controller.rb

```diff
@@ -85,5 +85,9 @@ def default_sort
     def flatpak_remote_params
       params.require(:flatpak_remote).permit(:name, :url, :description, :organization_id, :username, :token)
     end
+
+    def rejected_autocomplete_items
+      ['seeded', 'organization_id']
```

Comment:
Don't need this and if you remove `:complete_value => true,` from organization_id param in the controller, you can remove this entire section.

Reply:
Discussed offline..Don't need organization_id as autocompleted search param altogether but still need org_id as a search param so this is right..